src/command/boosterPass/revoke.ts

```diff
@@ -0,0 +1,84 @@
+import { MinehutCommand } from '../../structure/command/minehutCommand';
+import { PermissionLevel } from '../../util/permission/permissionLevel';
+import { Message } from 'discord.js';
+import { GuildMember } from 'discord.js';
+import { guildConfigs } from '../../guild/config/guildConfigs';
+import { BoosterPassModel } from '../../model/boosterPass';
+
+export default class BoosterPassGiveCommand extends MinehutCommand {
+	constructor() {
+		super('boosterpass-revoke', {
+			permissionLevel: PermissionLevel.NitroBooster,
+			// Only nitro boosters can revoke booster passes
+			enforcePermissionLevelRole: true,
+			category: 'boosterpass',
+			channel: 'guild',
+			description: {
+				content:
+					'Remove a booster pass from a member, if you have given them one',
+				usage: '<member>',
+				examples: [
+					'boosterpass revoke @Facto',
+					'boosterpass revoke 535986058991501323',
+				],
+			},
+			clientPermissions: ['MANAGE_ROLES'],
+			args: [
+				{
+					id: 'member',
+					type: 'member',
+					prompt: {
+						start: (msg: Message) =>
+							`${msg.author}, whose booster pass do you want to revoke?`,
+						retry: (msg: Message) => `${msg.author}, please mention a member.`,
+					},
+				},
+			],
+		});
+	}
+
+	async exec(msg: Message, { member }: { member: GuildMember }) {
+		const boosterPassConfiguration = guildConfigs.get(msg.guild!.id)?.features
```

Comment:
Since the guild config is retrieved a few times, it would be nicer if you just defined the `config` so the subsequent role and config definitions can just use `config`

```ts
const config = guildConfigs.get(msg.guild!.id);
const boosterPassConfig = config.features.boosterPass;
const nitroBoosterRole = config.roles.nitroBooster;
const boosterPassRole = config.roles.boosterPass;
```
This is just a rough idea of what I mean, this doesn't include the necessary `!` and `?`